dts/bindings/gpio/richtek,rt1718s-alert.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Richtek RS1718S TCPC chip alert handler
+
+    Example:
+      rt1718s_alert: rt1718s_alert {
```

Comment:
Hey do we really need a new device node for this? Seems like there's a 1:1 mapping, can't we reuse the original node with some extra properties instead?

Reply:
You are right. Done.